python/revng/pypeline/storage/util.py

```diff
@@ -2,4 +2,25 @@
 # This file is distributed under the MIT License. See LICENSE.md for details.
 #
 
+from revng.pypeline.object import Kind
+from revng.pypeline.utils.registry import get_singleton
+
+__all__ = ("_REVNG_VERSION_PLACEHOLDER", "_KIND_MAXDEPTH", "check_kind_structure")
+
 _REVNG_VERSION_PLACEHOLDER = "1.0.0"
+
+# Number of bytes that the biggest ObjectID will contain when serialized to bytes
+_KIND_MAXDEPTH = 17
```

Comment:
1. How did you come up with this number?
2. Why is this called "depth"? This is not a "depth".

Reply:
1. 1 byte for the Kind type + 16 bytes for the `MetaAddress`
2. It's depth in the sense that it's the maximum length in bytes of the longest sequence, see `MaxObjectSize` in REP49

Reply:
1. Done
2. Changed to `_OBJECTID_MAXSIZE`